spec/docs/participant_declarations_spec.rb

```diff
@@ -25,7 +25,7 @@
                 "type": "string",
               },
               "declaration_type": {
-                "enum": %w[started],
+                "enum": %w[started retained_1 retained_2 retained_3 retained_4 completed],
```

Comment:
Are we sure we want to add those to swagger schema?

Reply:
Only for consistency to show the applicable types since they also appear in the error messages when type is incorrect.